<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gameboard</class>
 <widget class="QMainWindow" name="Gameboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: qlineargradient(
        spread: pad,
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 rgba(0, 34, 133, 180),
        stop: 1 rgba(255, 255, 255, 80)
    );
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Reset_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: none;
    color: white;
    background-color: rgba(0, 120, 255, 180); /* أزرق شفاف */
    font: bold 11pt &quot;Arial&quot;;
    border-radius: 20px;
    padding: 8px;
}

QPushButton:hover {
    background-color: rgba(30, 144, 255, 220); /* أزرق أفتح عند المرور */
    font: bold 12pt &quot;Arial&quot;;
}

QPushButton:pressed {
    background-color: rgba(0, 80, 180, 255); /* أزرق داكن عند الضغط */
    font: bold 12pt &quot;Arial&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back_gboard">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>590</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: none;
    color: white;
    background-color: rgba(0, 120, 255, 180); /* أزرق شفاف */
    font: bold 11pt &quot;Arial&quot;;
    border-radius: 20px;
    padding: 8px;
}

QPushButton:hover {
    background-color: rgba(30, 144, 255, 220); /* أزرق أفتح عند المرور */
    font: bold 12pt &quot;Arial&quot;;
}

QPushButton:pressed {
    background-color: rgba(0, 80, 180, 255); /* أزرق داكن عند الضغط */
    font: bold 12pt &quot;Arial&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>561</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="p1_label">
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 rgba(30, 60, 114, 150),
        stop: 1 rgba(42, 82, 152, 150)
    );
    color: white;
    font: bold 14pt &quot;Segoe UI&quot;;
    padding: 14px;
    border-radius: 20px;
    border: 2px solid rgba(255, 255, 255, 180); 
    qproperty-alignment: AlignCenter;
}
QLabel[active=&quot;true&quot;] {
    background-color: qlineargradient(
        x1: 1, y1: 0,
        x2: 0, y2: 1,
        stop: 0 rgba(58, 28, 113, 180),
        stop: 1 rgba(215, 109, 119, 180)
    );
    color: #FFEB3B;
    border: 2px solid #FFEB3B;
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="another_label">
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 rgba(30, 60, 114, 150),
        stop: 1 rgba(42, 82, 152, 150)
    );
    color: white;
    font: bold 14pt &quot;Segoe UI&quot;;
    padding: 14px;
    border-radius: 20px;
    border: 2px solid rgba(255, 255, 255, 180); 
    qproperty-alignment: AlignCenter;
}
QLabel[active=&quot;true&quot;] {
    background-color: qlineargradient(
        x1: 1, y1: 0,
        x2: 0, y2: 1,
        stop: 0 rgba(58, 28, 113, 180),
        stop: 1 rgba(215, 109, 119, 180)
    );
    color: #FFEB3B;
    border: 2px solid #FFEB3B;
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="c_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Adobe Express - file.png</normaloff>../Downloads/Adobe Express - file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>125</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="c_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Adobe Express - file.png</normaloff>../Downloads/Adobe Express - file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>125</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="c_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Adobe Express - file (1).png</normaloff>../Downloads/Adobe Express - file (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="c_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Adobe Express - file (1).png</normaloff>../Downloads/Adobe Express - file (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>125</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="c_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Adobe Express - file.png</normaloff>../Downloads/Adobe Express - file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="c_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Adobe Express - file (1).png</normaloff>../Downloads/Adobe Express - file (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>125</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="c_9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Adobe Express - file.png</normaloff>../Downloads/Adobe Express - file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>125</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="c_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Adobe Express - file.png</normaloff>../Downloads/Adobe Express - file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="c_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
    border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Adobe Express - file (1).png</normaloff>../Downloads/Adobe Express - file (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>125</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="upper">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <fontweight>Thin</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 1); /* Neon white with 30% opacity */
    border: none;
    min-width: 0px;
    min-height: 0px;
    border-radius: 7px;
    box-shadow: 0 0 4px rgba(255, 255, 255, 1); /* Lighter neon glow */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Lower">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 1); /* Neon white with 30% opacity */
    border: none;
    min-width: 0px;
    min-height: 0px;
    border-radius: 7px;
    box-shadow: 0 0 4px rgba(255, 255, 255, 1); /* Lighter neon glow */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Right">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>139</y>
      <width>16</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    background-color: rgba(255, 255, 255, 1); /* Neon white with 30% opacity */
    border: none;
    min-width: 0px;
    min-height: 0px;
    border-radius: 7px;
    box-shadow: 0 0 4px rgba(255, 255, 255, 1); /* Lighter neon glow */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="left">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>140</y>
      <width>16</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#left {
    background-color: rgba(255, 255, 255, 1); /* Neon white with 30% opacity */
    border: none;
    min-width: 0px;
    min-height: 0px;
    border-radius: 7px;
    box-shadow: 0 0 4px rgba(255, 255, 255, 1); /* Lighter neon glow */
}</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="undo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>590</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: none;
    color: white;
    background-color: rgba(0, 120, 255, 180); /* أزرق شفاف */
    font: bold 11pt &quot;Arial&quot;;
    border-radius: 20px;
    padding: 8px;
}

QPushButton:hover {
    background-color: rgba(30, 144, 255, 220); /* أزرق أفتح عند المرور */
    font: bold 12pt &quot;Arial&quot;;
}

QPushButton:pressed {
    background-color: rgba(0, 80, 180, 255); /* أزرق داكن عند الضغط */
    font: bold 12pt &quot;Arial&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>70</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="p1_wins_label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Shared style for all 3 states */
QLabel#p1_wins_label[status=&quot;win&quot;],
QLabel#p1_wins_label[status=&quot;lose&quot;],
QLabel#p1_wins_label[status=&quot;draw&quot;] {
    border-radius: 10px;
    font-weight: bold;
    padding: 6px;
    qproperty-alignment: AlignCenter;  /* Center the text */
    min-width: 100px; /* Optional: helps keep shape */
    min-height: 30px;
}

/* 🟢 Win = Neon Green */
QLabel#p1_wins_label[status=&quot;win&quot;] {
    background-color: rgba(0, 255, 0, 50);
    border: 2px solid rgba(0, 255, 0, 180);
    color: white;
}

/* 🔴 Lose = Neon Red */
QLabel#p1_wins_label[status=&quot;lose&quot;] {
    background-color: rgba(255, 0, 0, 50);
    border: 2px solid rgba(255, 0, 0, 180);
    color: white;
}

/* 🟡 Draw = Neon Yellow */
QLabel#p1_wins_label[status=&quot;draw&quot;] {
    background-color: rgba(255, 255, 0, 50);
    border: 2px solid rgba(255, 255, 0, 180);
    color: white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="ties_label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: rgba(255, 255, 0, 50);  /* Neon yellow with transparency */
    color: white;                             /* Text color */
    border: 2px solid rgba(255, 255, 0, 180); /* Neon border */
	qproperty-alignment: AlignCenter;  /* Center the text */
    border-radius: 10px;
    padding: 5px;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="p2_wins_label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Shared style for all 3 states */
QLabel#p2_wins_label[status=&quot;win&quot;],
QLabel#p2_wins_label[status=&quot;lose&quot;],
QLabel#p2_wins_label[status=&quot;draw&quot;] {
    border-radius: 10px;
    font-weight: bold;
    padding: 6px;
    qproperty-alignment: AlignCenter;  /* Center the text */
    min-width: 100px; /* Optional: helps keep shape */
    min-height: 30px;
}

/* 🟢 Win = Neon Green */
QLabel#p2_wins_label[status=&quot;win&quot;] {
    background-color: rgba(0, 255, 0, 50);
    border: 2px solid rgba(0, 255, 0, 180);
    color: white;
}

/* 🔴 Lose = Neon Red */
QLabel#p2_wins_label[status=&quot;lose&quot;] {
    background-color: rgba(255, 0, 0, 50);
    border: 2px solid rgba(255, 0, 0, 180);
    color: white;
}

/* 🟡 Draw = Neon Yellow */
QLabel#p2_wins_label[status=&quot;draw&quot;] {
    background-color: rgba(255, 255, 0, 50);
    border: 2px solid rgba(255, 255, 0, 180);
    color: white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
